tests/test_differentiation.py

```diff
@@ -65,6 +65,54 @@ def test_adjoint_diff(n_qubits: int, n_layers: int) -> None:
     # )
 
 
+@pytest.mark.parametrize("n_qubits", [3])
+@pytest.mark.parametrize("batch_size", [1])
```

Comment:
Is it worth parametrising for a single value ?

Reply:
Oups, forgot to put more values back...